src/sentry/event_manager.py

```diff
@@ -719,28 +719,6 @@ def save(self, project_id, raw=False, assume_normalized=False):
         # store a reference to the group id to guarantee validation of isolation
         event.data.bind_ref(event)
 
-        # When an event was sampled, the canonical source of truth
-        # is the EventMapping table since we aren't going to be writing out an actual
-        # Event row. Otherwise, if the Event isn't being sampled, we can safely
-        # rely on the Event table itself as the source of truth and ignore
-        # EventMapping since it's redundant information.
-        if is_sample:
```

Comment:
projects:sample-events feature is still true by default in server.py. Did you double check none is relying on EventMapping anymore even if the event is in Snuba ?

Reply:
I've removed EventMapping everywhere it's used in the codebase except the deletions code (will follow up with that). I think we might also be wanting to turn off this feature switch completely as a follow up as well, but I'll need to double check on that.

Reply:
```
(sentry) bash-3.2$ git grep EventMapping                                                                         
src/sentry/api/endpoints/organization_eventid.py:from sentry.models import Project, Event, EventMapping
src/sentry/api/endpoints/organization_eventid.py:                event_mapping = EventMapping.objects.filter(event_id=event_id,
src/sentry/deletions/__init__.py:    default_manager.register(models.EventMapping, BulkModelDeletionTask)
src/sentry/deletions/defaults/event.py:            ModelRelation(models.EventMapping, key),
src/sentry/deletions/defaults/group.py:            models.EventMapping,
src/sentry/deletions/defaults/project.py:            models.Activity, models.EnvironmentProject, models.EventAttachment, models.EventMapping,
src/sentry/event_manager.py:    Activity, Environment, Event, EventDict, EventError, EventMapping, EventUser, Group,
src/sentry/event_manager.py:        # isn't a perfect solution -- this doesn't handle ``EventMapping`` and
src/sentry/event_manager.py:        # is the EventMapping table since we aren't going to be writing out an actual
src/sentry/event_manager.py:        # EventMapping since it's redundant information.
src/sentry/event_manager.py:                with transaction.atomic(using=router.db_for_write(EventMapping)):
src/sentry/event_manager.py:                    EventMapping.objects.create(project=project, group=group, event_id=event_id)
src/sentry/event_manager.py:                        'model': EventMapping.__name__,
src/sentry/models/eventmapping.py:class EventMapping(Model):
src/sentry/runner/commands/cleanup.py:                models.EventMapping,
src/sentry/runner/commands/cleanup.py:        (models.EventMapping, 'date_added', '-date_added'),
```

What about . organization_eventid.py and src/sentry/runner/commands/cleanup.py

Reply:
`organization_eventid.py` has been updated already. I will open a follow up PR to this to tackle the cleanup command and deletions.

Reply:
I think there is one in merge.py that needs updating now